crates/install-wheel-rs/src/wheel.rs

```diff
@@ -237,8 +237,11 @@ fn get_script_executable(python_executable: &Path, is_gui: bool) -> PathBuf {
     // Only check for pythonw.exe on Windows
     if cfg!(windows) && is_gui {
         python_executable
-            .parent()
-            .map(|parent| parent.join("pythonw.exe"))
+            .file_name()
```

Comment:
Might be useful to include the pip reference to [_get_alternate_executable](https://github.com/pypa/pip/blob/76e82a43f8fb04695e834810df64f2d9a2ff6020/src/pip/_vendor/distlib/scripts.py#L121-L126)

e.g.

 ```rust
/// Returns a [`PathBuf`] to `python[w].exe` for script execution.
///
/// <https://github.com/pypa/pip/blob/76e82a43f8fb04695e834810df64f2d9a2ff6020/src/pip/_vendor/distlib/scripts.py#L121-L126>
 ```

Reply:
Added.